images/li/sle12_sp3/root/usr/lib/systemd/network/81-mac-link

```diff
@@ -0,0 +1,2 @@
+[Link]
+NamePolicy=mac
```

Comment:
I think it should end with a newline

Reply:
along with the policy change the files ```./usr/lib/udev/rules.d/81-net-setup-link.rules``` needs to be deleted from the image descriptions

Reply:
Done